doc/roadmap.rst

```diff
@@ -82,7 +82,7 @@ We think the right approach to extending xarray's user community and the
 usefulness of the project is to focus on improving key interfaces that
 can be used externally to meet domain-specific needs.
 
-We can generalize the community's needs into three main catagories:
+We can generalize the community's needs into three main caeagories:
```

Comment:
shouldn't this be
```suggestion
We can generalize the community's needs into three main categories:
```
?

Since this PR has already been merged we would need a new PR. @slowy07, would you be up for that?

Reply:
okay sir @keewis ,i'll do it,thanks for the review